packages/db/README.md

```diff
@@ -1,11 +1,11 @@
-# `truffle-db`
+# `db`
```

Comment:
Same here

(**Note**: the directory name is correct as `packages/db`)

Reply:
(Here's one you missed because my comment was confusing - we **do** want the README to say the full package name)